chain/client/src/test_utils.rs

```diff
@@ -57,14 +55,38 @@ use near_network::types::{NetworkInfo, PeerManagerMessageRequest, PeerManagerMes
 use near_network_primitives::types::{
     AccountOrPeerIdOrHash, NetworkViewClientMessages, NetworkViewClientResponses,
     PartialEncodedChunkRequestMsg, PartialEncodedChunkResponseMsg, PeerChainInfoV2, PeerInfo,
+    SetChainInfo,
 };
 use near_primitives::epoch_manager::RngSeed;
 use near_primitives::network::PeerId;
 use near_primitives::runtime::config::RuntimeConfig;
 use near_primitives::time::{Clock, Instant};
 use near_primitives::utils::MaybeValidated;
 
-pub type PeerManagerMock = Mocker<PeerManagerActor>;
```

Comment:
:+1: :+1: :+1:

Reply:
though, this could've been split into a separate PR

Reply:
extracted to separate PR